### ISSUE TITLE
Mac: Re-enable GitCommands.StatusTests.MoveFileIntoDotGitDirectory

### ISSUE THREAD

**Issue by wilbaker** [microsoft/VFSForGit#312]
Two reports so far.

Example:
https://dev.azure.com/mseng/VSOnline/_build/results?buildId=7366737&view=logs

```
2018-09-27T23:46:09.3543540Z 1-1) Failed : GVFS.FunctionalTests.Tests.GitCommands.StatusTests.MoveFileIntoDotGitDirectory()
2018-09-27T23:46:09.3543960Z status Output Lines
2018-09-27T23:46:09.3544170Z status Output Lines counts do not match. was: 2 expected: 4
2018-09-27T23:46:09.3544430Z Extra: nothing to commit, working tree clean
2018-09-27T23:46:09.3544670Z Missing: Changes not staged for commit:
2018-09-27T23:46:09.3544890Z Missing:   deleted: Readme.md
2018-09-27T23:46:09.3545080Z Missing: no changes added to commit
2018-09-27T23:46:09.3545410Z at GVFS.Tests.Should.EnumerableShouldExtensions.ShouldMatchInOrder[T](IEnumerable`1 group, IEnumerable`1 expectedValues, Func`3 equals, String message) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.Tests/Should/EnumerableShouldExtensions.cs:line 119
2018-09-27T23:46:09.3546260Z at GVFS.FunctionalTests.Tools.GitHelpers.ValidateGitCommand(GVFSFunctionalTestEnlistment enlistment, ControlGitRepo controlGitRepo, String command, Object[] args) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tools/GitHelpers.cs:line 90
2018-09-27T23:46:09.3546700Z at GVFS.FunctionalTests.Tests.GitCommands.GitRepoTests.ValidateGitCommand(String command, Object[] args) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs:line 178
2018-09-27T23:46:09.3547100Z at GVFS.FunctionalTests.Tests.GitCommands.StatusTests.MoveFileIntoDotGitDirectory() in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs:line 23
2018-09-27T23:46:09.3547340Z 
2018-09-27T23:46:09.3548820Z 1-2) Failed : GVFS.FunctionalTests.Tests.GitCommands.StatusTests.MoveFileIntoDotGitDirectory()
2018-09-27T23:46:09.3548990Z status Output Lines
2018-09-27T23:46:09.3549120Z status Output Lines counts do not match. was: 2 expected: 4
2018-09-27T23:46:09.3549250Z Extra: nothing to commit, working tree clean
2018-09-27T23:46:09.3549370Z Missing: Changes not staged for commit:
2018-09-27T23:46:09.3549480Z Missing:   deleted: Readme.md
2018-09-27T23:46:09.3549580Z Missing: no changes added to commit
2018-09-27T23:46:09.3549780Z at GVFS.Tests.Should.EnumerableShouldExtensions.ShouldMatchInOrder[T](IEnumerable`1 group, IEnumerable`1 expectedValues, Func`3 equals, String message) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.Tests/Should/EnumerableShouldExtensions.cs:line 119
2018-09-27T23:46:09.3550060Z at GVFS.FunctionalTests.Tools.GitHelpers.ValidateGitCommand(GVFSFunctionalTestEnlistment enlistment, ControlGitRepo controlGitRepo, String command, Object[] args) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tools/GitHelpers.cs:line 90
2018-09-27T23:46:09.3550330Z at GVFS.FunctionalTests.Tools.GitHelpers.ValidateGitCommand(GVFSFunctionalTestEnlistment enlistment, ControlGitRepo controlGitRepo, String command, Object[] args) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tools/GitHelpers.cs:line 95
2018-09-27T23:46:09.3550600Z at GVFS.FunctionalTests.Tests.GitCommands.GitRepoTests.ValidateGitCommand(String command, Object[] args) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs:line 178
2018-09-27T23:46:09.3550830Z at GVFS.FunctionalTests.Tests.GitCommands.GitRepoTests.CheckHeadCommitTree() in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs:line 139
2018-09-27T23:46:09.3551390Z at GVFS.FunctionalTests.Tests.GitCommands.GitRepoTests.TestValidationAndCleanup(Boolean ignoreCase) in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs:line 90
2018-09-27T23:46:09.3551640Z at GVFS.FunctionalTests.Tests.GitCommands.GitRepoTests.TearDownForTest() in /Users/vsts/agent/2.140.2/work/1/s/GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs:line 83
```

**Comment by wilbaker** [comment]
Could be related to #315 (e.g. if the user-mode was collapsing two different notifications together)

**Comment by wilbaker** [comment]
See also https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=7424273&view=logs.  This build had the fix for #315, and so this is probably something else.

**Comment by wilbaker** [comment]
This issue will most likely require #395 and\or #396 (to root cause why VFS4G occasionally misses notifications).

CC: @jrbriggs @sanoursa as FYI

**Comment by wilbaker** [comment]
Renaming this issue as I will be disabling the test for now.

**Comment by wilbaker** [comment]
Removed "test reliability" label as #456 disables this test for now.

**Comment by wilbaker** [comment]
Fixed in #1166
